client/app/templates/components/project-milestone.hbs

```diff
@@ -1,23 +1,9 @@
 <div class="cell shrink">
-  {{#if (eq milestone.statuscode "Completed")}}
-    <span class="milestone-icon fa-layers fa-fw">
-      {{fa-icon icon='circle' fixedWidth=true class='dark-gray'}}
-      {{fa-icon icon='check' fixedWidth=true class='blue-light' transform='shrink-6'}}
-    </span>
-  {{else if (eq milestone.statuscode "In Progress")}}
-    <span class="milestone-icon fa-layers fa-fw">
-      {{fa-icon icon='circle' fixedWidth=true class='green-dark'}}
-      {{fa-icon icon='calendar' prefix='far' fixedWidth=true class='white-smoke' transform='shrink-8'}}
-    </span>
-  {{else}}{{!-- milestone.statuscode "Not Started" --}}
-    <span class="milestone-icon fa-layers fa-fw">
-      {{fa-icon icon='circle' fixedWidth=true class='light-gray'}}
-      {{fa-icon icon='calendar' prefix='far' fixedWidth=true class='off-white' transform='shrink-8'}}
-    </span>
-  {{/if}}
+  <span class="milestone-icon fa-layers fa-fw">
+  </span>
 </div>
```

Comment:
What's this span for?

Reply:
the fa-layeres and fa-fw helps with spacing and indent lol 🙈 😬